base_tier_validation/tests/test_tier_validation.py

```diff
@@ -0,0 +1,1027 @@
+# Copyright 2018-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright (c) 2022 brain-tec AG (https://braintec.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import tagged
+
+from .common import CommonTierValidation
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidation(CommonTierValidation):
+    # def test_01_auto_validation(self):
```

Comment:
why are you comment these test cases?
Are you trying to cheat the tests?

Reply:
bad manipulation, removed